docs/content/en/docs/operator-manual/piped/installation/installing-on-google-cloud-vm.md

```diff
@@ -0,0 +1,98 @@
+
+## Installation
+
+- Preparing a piped configuration file as the following:
+
+  ``` yaml
+  apiVersion: pipecd.dev/v1beta1
+  kind: Piped
+  spec:
+    projectID: {PROJECT_ID}
+    pipedID: {PIPED_ID}
+    pipedKeyData: {BASE64_ENCODED_PIPED_KEY}
+    webAddress: {CONTROL_PLANE_WEB_ADDRESS}
+    # Write in a format like "host:443" because the communication is done via gRPC.
+    apiAddress: {CONTROL_PLANE_API_ADDRESS}
+
+    git:
+      sshKeyData: {BASE64_ENCODED_PRIVATE_SSH_KEY}
+
+    repositories:
+      - repoId: {REPO_ID_OR_NAME}
+        remote: git@github.com:{GIT_ORG}/{GIT_REPO}.git
+        branch: {GIT_BRANCH}
+
+    # Optional
+    # Uncomment this if you want to enable this Piped to handle CloudRun application.
+    # cloudProviders:
+    #  - name: cloudrun-in-project
+    #    type: CLOUDRUN
+    #    config:
+    #      project: {GCP_PROJECT_ID}
+    #      region: {GCP_PROJECT_REGION}
+
+    # Optional
+    # Uncomment this if you want to enable this Piped to handle Terraform application.
+    #  - name: terraform-gcp
+    #    type: TERRAFORM
+
+    # Optional
+    # Uncomment this if you want to enable SecretManagement feature.
+    # https://pipecd.dev/docs/user-guide/secret-management
+    # secretManagement:
+    #   type: KEY_PAIR
+    #   config:
+    #     privateKeyData: {BASE64_ENCODED_PRIVATE_KEY}
+    #     publicKeyData: {BASE64_ENCODED_PUBLIC_KEY}
+  ```
+
+- Creating a new secret in [SecretManager](https://cloud.google.com/secret-manager/docs/creating-and-accessing-secrets) to store above configuration data securely
+
+  ``` console
+  gcloud secrets create vm-piped-config --data-file={PATH_TO_CONFIG_FILE}
+  ```
+
+- Creating a new Service Account for Piped and giving it the ability to access above secret
+
+  ``` console
+  gcloud iam service-accounts create vm-piped \
+    --description="Using by Piped running on Google Cloud VM" \
+    --display-name="vm-piped"
+
+  gcloud secrets add-iam-policy-binding vm-piped-config \
+    --member="serviceAccount:vm-piped@{GCP_PROJECT_ID}.iam.gserviceaccount.com" \
+    --role="roles/secretmanager.secretAccessor"
+  ```
+
+- Running Piped on a Google Cloud VM
+
+  ``` console
+  gcloud compute instances create-with-container vm-piped \
+    --container-image="gcr.io/pipecd/piped:{{< blocks/latest_version >}}" \
+    --container-arg="piped" \
+    --container-arg="--config-gcp-secret=projects/{GCP_PROJECT_NUMBER}/secrets/vm-piped-config/versions/{SECRET_VERSION}" \
+    --network="{VPC_NETWORK}" \
+    --subnet="{VPC_SUBNET}" \
+    --scopes="https://www.googleapis.com/auth/cloud-platform" \
+    --service-account="vm-piped@{GCP_PROJECT_ID}.iam.gserviceaccount.com"
+  ```
+
+After that, you can see on PipeCD web at `Settings` page that Piped is connecting to control-plane.
```

Comment:
Not sure do we have the ability to watch the connecting status of piped on Setting page currently 👀

Reply:
Yes, we can see when it was started.

Reply:
Ah, I remembered that 👍